openrtb_ext/bid_response_video.go

```diff
@@ -0,0 +1,26 @@
+package openrtb_ext
+
+type BidResponseVideo struct {
+	AdPods []*AdPod `json:"adPods"`
+}
+
+type AdPod struct {
+	PodId     int64            `json:"podid"`
+	Targeting []VideoTargeting `json:"targeting"`
+	Errors    VideoErrors      `json:"errors"`
+}
+
+type VideoTargeting struct {
+	Hb_pb         string `json:"hb_pb"`
+	Hb_pb_cat_dur string `json:"hb_pb_cat_dur"`
+	Hb_cache_id   string `json:"hb_cache_id"`
+}
+
+type VideoErrors struct {
```

Comment:
We might want this to be a map[string]json.RawMessage since OpenX isn't the only bidder that may return errors. I'm not totally sure though if there is a consistent way it's done like if we can define a struct for the error messages that all bidders abide by or if we need to just allow it to be a generic json blob.